### ISSUE TITLE
[BUG] Dropdown exceeds viewport in select widths.

### ISSUE THREAD

**Issue by troelslenda** [kirbydesign/designsystem#3104]
- [ x ] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the bug
When the viewport hits width somewhat between 320 <-> 390 px the dropdown renders out of the viewport. In some cases the checkmark that shows the selected item is completely invisible, which makes it difficult to identify the selected item. See enclosed screenshot 
 

## Describe how to reproduce the bug
1. Visit https://cookbook.kirby.design/#/home/showcase/dropdown
2. Set the viewport to 350px wide
3. Open the Default / Simple dropdown
4. select one item.
5. Open the same dropdown again.

## Which Kirby version was used?
 
8.6.0

## What was the expected behavior?
That the dropdown resides within the viewport.

## Add any screenshots

![Screenshot 2023-06-14 145229](https://github.com/kirbydesign/designsystem/assets/263060/b4ebfac0-ab79-43ae-86c7-ff5072ba4b49)

## Please complete the following information:
- OS: Windows 11
- Browsers:
  - Vivaldi 6.1.3035.75 (Chromium)  
  - Edge 114.0.1823.43

*Also reported by third-party on Android*

## Are there any additional context?

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/main/.github/SUPPORT.md#team-kirby); do not proceed until this is done

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.

**Comment by jakobe** [comment]
@troelslenda Thanks for reporting 👍🏻 

I can't recreate this though - when I resize the viewport to 320px width it looks fine:
![image](https://github.com/kirbydesign/designsystem/assets/9556693/6cd677e6-0a14-448b-a574-7036e4f8cee8)

**Comment by troelslenda** [comment]
It's not that noticeable in cookbook - I agree with that. But nevertheless try to resize to 350 as I initially reported. The issue was never there with a 320px viewport.

**Comment by RasmusKjeldgaard** [comment]
Closing this issue automatically because it has been open for more than 6 months and is not currently in progress. If this is still relevant, please feel free to reopen it or open a new issue with updated details. Thank you!
